site/src/component/SearchModule/SearchModule.tsx

```diff
@@ -110,7 +110,7 @@ const SearchModule: FC<SearchModuleProps> = ({ index }) => {
     if (pendingRequest) {
       clearTimeout(pendingRequest);
     }
-    const timeout = setTimeout(() => {
+    const timeout = window.setTimeout(() => {
```

Comment:
Is this for consistency? I'm not sure if we do this elsewhere in the codebase?

Reply:
Node/typescript is dumb and half the time setTimeout's return type is NodeJS.Timeout, the other half it's number and this causes an error when it doesn't match the type of the state for pending request. window.setTimeout always has a consistent return type of NodeJS.Timeout for some reason so I changed it to that.